### PR TITLE
Added icon for NameColor extension

### PR DESCRIPTION
**Pull request by oxxie**
As suggested by @mesqueeb with #1127.

### CODE REVIEW

**Comment by mesqueeb** [comment]
@oxxie Nice!!

@invariant I can confirm this icon is royalty free, because I'm the one who designed that icon 😄

**Review by invariant — APPROVED** [review]

**Comment by invariant** [comment]
Thanks both of you for this. In the Config.plist the icon actually has to go in the Image File key in the Action dict, rather than Extension Image File. I'll make the changes.

**Review by invariant — APPROVED** [review]

**Comment by invariant** [comment]
Merged in 5d40481a95993c1d9e9b3d188b3db73999cfc4e2

**Comment by mesqueeb** [comment]
@invariant will the be deployed to the docs website soon ? Wanna send it to my friends.

**Comment by invariant** [comment]
I'll have to think about that, I don't put them all on the site main site. Normally I only put the most highly requested on on there so as not to flood it with extensions. But your enthusiasm is noted .... can you sell it to me, tell me a bit more about what the point of it is :)

BTW you can send this link: https://github.com/pilotmoon/PopClip-Extensions/blob/master/extensions/NameColor.popclipextz?raw=true although I realise it is not as nice as the proper download page.

**Comment by mesqueeb** [comment]
@invariant Sure!
So I'm a front-end developer and a lot of my colleagues and friends are as well.
The reason we want to use this is because every time we introduce colours in our websites and apps, we basically replace that colour with a variable name, so we only have to define the colour code once and can use the variable name everywhere.

However, instead of very _generic_ variable names like `primary` `secondary` etc. it is easier for the mind to link the colour with an original name.
Some names we've used in websites and apps for colour variable names:
- stone
- carbon
- dandelion

etc.

Coming up with a good name for the colour is fun, but most of the time it's something we use that tool for as I shared before, called **"name that color"**:
http://chir.ag/projects/name-that-color/#6195ED

This PopClip extension basically replaces the need to open that website and paste in the hex value we want to use to come up with a colour name we can use for variable.

Long story short: it's a must have for any front-end developer! 🎉

**Comment by mesqueeb** [comment]
@invariant any thoughts on my argument above that it's a great extension for designers and front end developers? :O

Would love to hear your thoughts!!

**Comment by mesqueeb** [comment]
@pilotmoon Maybe we can have a "see more" section on the new popclip website at the bottom of each category.

In the "developers" category we'd see it and clicking it will reveal this **Name This Color** extension.

eoueou

**Comment by pilotmoon** [comment]
I'm working on a new backend and extension submission mechanism to allow lost more user-contributed extensions in the directory. So you'll be able to add it soon. It just will take a little while, but it is coming!
